### PR TITLE
Updating to look like en_US.lang lattest format

### PR DESCRIPTION
**Pull request by efimov90**
some string work:
sorting like en_EN, removing, updating some strings to new strings, partial translating

проведена работа со строками:
перемещение, замена неработающих строк на обновленные аналоги, удаление лишних строк, частичный перевод дополняющих строк.

### CODE REVIEW

File: assets/railcraft/lang/ru_RU.lang [hunk 1]
```diff
@@ -394,44 +473,30 @@ railcraft.gui.help=Помощь
...
-railcraft.gui.item.loader.manual.tip=Никогда не излучать сигнал крас. камня
-railcraft.gui.item.loader.manual=Вручную
-railcraft.gui.item.loader.stock.tip=Перемещать предметы\n до совпадения с фильтром
-railcraft.gui.item.loader.stock=Содержимое
-railcraft.gui.item.loader.transfer.tip=Переместить в точности столько предметов,\n сколько их в фильтре
-railcraft.gui.item.loader.transfer=Количество
+gui.railcraft.manipulator.buffer=Буфер
+gui.railcraft.manipulator.redstone.complete.name=Завершено
+gui.railcraft.manipulator.redstone.complete.tips=Продолжать до конца операции
+gui.railcraft.manipulator.redstone.immediate.name=Immediate
```
Comment (liach) [inline assets/railcraft/lang/ru_RU.lang]: Can you translate leftover entries like these?

Comment (efimov90) [reply assets/railcraft/lang/ru_RU.lang]: Sure, but not very fast. I will continue translating.

Comment (liach) [reply assets/railcraft/lang/ru_RU.lang]: Will wait then

Comment (efimov90) [reply assets/railcraft/lang/ru_RU.lang]: Can you please explain what exact is Charge Trap? 715 tile.railcraft.charge.trap.name=Charge Trap

File: assets/railcraft/lang/ru_RU.lang [hunk 2]
```diff
@@ -1073,3 +1156,113 @@ tile.railcraft.wall.stone.brick.name=Стена из каменного кирп
...
+track_kit.railcraft.railcraft_routing.tips=Устанавливает проходящему локомотиву билет\n§o§9- Ударьте монтировкой, чтобы изменить билет -
+
+track_kit.railcraft.railcraft_transition.name=Переход
+track_kit.railcraft.railcraft_transition.tips=Переводит на/с Высокой скорости\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_whistle.name=Активатор свистка
+track_kit.railcraft.railcraft_whistle.tips=Проходящие локомотивы\n подуют в свисток\n§o§c- Необходим источник Redstone-сигнала -
+
+track_type.railcraft.railcraft_iron.name=Железный
+track_type.railcraft.railcraft_strap_iron.name=Strap Iron
```
Comment (liach) [inline assets/railcraft/lang/ru_RU.lang]: This is old wooden track

File: assets/railcraft/lang/ru_RU.lang [hunk 3]
```diff
@@ -1073,3 +1156,113 @@ tile.railcraft.wall.stone.brick.name=Стена из каменного кирп
...
+track_type.railcraft.railcraft_abandoned.name=Заброшенный
+track_type.railcraft.railcraft_reinforced.name=Усилинный
+track_type.railcraft.railcraft_high_speed.name=H. S.
```
Comment (liach) [inline assets/railcraft/lang/ru_RU.lang]: This one means "High Speed" and below is "High Speed Electric"

Comment (efimov90) [reply assets/railcraft/lang/ru_RU.lang]: Ok, can i use full size words? Or better make like in en_us with 1st letters of words?

**Comment by efimov90** [comment]
ok, so i translated traps right.

File: assets/railcraft/lang/ru_RU.lang (hunk 3 above)
Comment (liach) [reply assets/railcraft/lang/ru_RU.lang]: Either should be fine

**Comment by liach** [comment]
Can I merge it now?

**Review by liach — APPROVED** [review]

**Comment by efimov90** [comment]
Yes, i think this patch is ready now.

**Pull request merged**